bucket/user_agent.go

```diff
@@ -6,10 +6,10 @@ import (
 	"io"
 	"net/http"
 	"sort"
-	"sync"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/mateusz/tempomat/lib/config"
```

Comment:
Tossed the config into a idomatic lib folder so that the buckets can get the type for Buckatable.SetConfig(config.Config)